packages/opentelemetry-api/src/trace/span_context.ts

```diff
@@ -50,6 +50,25 @@ export interface SpanContext {
    * SAMPLED = 0x1 and NONE = 0x0;
    */
   traceFlags: TraceFlags;
+  /**
+   * Debug flag to propagate.
+   *
+   * This flag is relevant to b3. According to the b3 spec,
+   * Debug is encoded as X-B3-Flags: 1. Absent or any other value can be ignored.
+   * Debug is a production troubleshooting aid used in tools like curl or chrome debug.
+   * Debug is an emphasized accept decision that implies accept (or setting traceFlags to accept),
+   * additionally reporting Span.debug = true for each span in the trace. Adding this
+   * to the span context because the b3 spec says when this flag is set properly,
+   * do not send the sampled header as well.
+   */
+  debug?: boolean;
```

Comment:
Do not add non-specified fields to the trace context. If you need to propagate an option, the context or perhaps the tracestate is the place to do that.